subjects/java/checkpoints/top-frequents/README.md

```diff
@@ -0,0 +1,55 @@
+## Top Frequents
+
+### Instructions
+
+Given a non-empty array of integers, return the k most frequent elements. Use a hash map and a heap for efficient lookup and retrieval.
```

Comment:
What if k is greater than the number of unique elements in the array? Maybe adding an example to show what's expected could be enough

Reply:
okay i will add a test for it